test/unit/testcore.js

```diff
@@ -1,9 +0,0 @@
-import {PhiloGL} from '../../src';
-import test from 'tape';
```

Comment:
I think a few basic test cases for the Utils would be very quick to write and well worth it